quickstart/install-terraform.py

```diff
@@ -35,14 +42,19 @@ def terraform_install(version=TERRAFORM_VERSION):
 
 
 if __name__ == '__main__':
+    # Set up argparse
+    parser_description = ('Installs Terraform in the specified directory.')
 
+    parser = argparse.ArgumentParser(description=parser_description)
+    parser.add_argument('dir', help='specify the directory to install Terraform')
+    args = parser.parse_args()
+
+    TERRAFORM_BIN_DIR = args.dir
```

Comment:
Yes, the command line argument would override the global bin dir. The script's default is the same as what we pass in at the moment, is this ok?

Reply:
Is there a scenario when the value assigned in L18 is used then?

Reply:
Currently L18 would not be used - should it be initialized to None instead?

Or should I make the argparse directory argument an optional one? That way if no argument is specified to insta--terraform.py, then it would use the default directory in the script. Which way is better?

Reply:
How about remove the global, make it a required parameter for this file, and pass it into the terraform_install() function as a required parameter.